unittests/emitc_tosa.cpp

```diff
@@ -266,4 +266,57 @@ TEST(tosa, reshape) {
   EXPECT_THAT(s3, Pointwise(Eq(), t3));
 }
 
+TEST(tosa, transpose) {
+  // clang-format off
+  Tensor2D<float, 3, 2> t0 = {1, 2,
+                              3, 4,
+                              5, 6};
+  Tensor1D<int32_t, 2> perms_i32 = {1, 0};
+  Tensor1D<int64_t, 2> perms_i64 = {1, 0};
+  Tensor1D<int64_t, 2> no_perms =  {0, 1};
+  Tensor2D<float, 2, 3> expected_result0 = {1, 3, 5,
+
```

Comment:
```suggestion

```
Is this empty line on purpose?

Reply:
No, that happens when you turn of clang-format :)